namer/controller.go

```diff
@@ -24,7 +24,7 @@ type (
 		Get(name string) (*VersionedDtab, error)
 		Create(name string, dtabstr string) (Version, error)
 		Delete(name string) error
-		Update(name string, dtabstr string, version Version) (Version, error)
+		Update(name string, dtabstr string) (Version, error)
```

Comment:
It seems like we'd still want update to take a version -- we just want another method Put that doesn't?  or we want to support nil versions?

We should think of this as a pure API client independent of the CLI.